core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -363,6 +363,16 @@ public <T extends RESTResponse> T handleRequest(
     return null;
   }
 
+  private void dropNamespace(Map<String, String> vars) {
+    Namespace namespace = namespaceFromPathVars(vars);
+    boolean cascade = PropertyUtil.propertyAsBoolean(vars, "cascade", false);
+    if (cascade) {
```

Comment:
Couldn't we always just call the three arg version here? since we can just call CatalogHandlers.dropNamespace(asNamesapceCatalog, namespace, cascade)

Reply:
yeah, that works. Fixed, Thanks